src/forms/FormFieldAddress/AddressAutocomplete/AddressAutocompleteTypes.ts

```diff
@@ -0,0 +1,21 @@
+import { TextFieldProps } from "@mui/material/TextField";
+
+export interface AddressAutocompleteProps {
+	value: string;
+	onChange: (address: string) => void;
+	className?: string;
+	onSelect: (value: any) => Promise<void>;
+	textField?: TextFieldProps;
+	fieldSize?: string;
+	placeholder?: string
+}
+
+export interface InputSettings {
```

Comment:
This doesn't seem to be used anywhere else in the project, also... it's the same interface as AddressAutocompleteProps, so we should just have 1